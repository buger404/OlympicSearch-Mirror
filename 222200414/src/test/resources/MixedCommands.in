totals
tOtal
result0810
result 1312
result sss
result 116
result 0130
result Q5
total
PK 中国 美利坚合众国 霹雳舞