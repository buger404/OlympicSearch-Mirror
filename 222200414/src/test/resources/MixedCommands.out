Error
-----
Error
-----
Error
-----
N/A
-----
N/A
-----
N/A
-----
N/A
-----
N/A
-----
rank1:美利坚合众国
gold:40
silver:44
bronze:42
total:126
-----
rank2:中国
gold:40
silver:27
bronze:24
total:91
-----
rank3:日本
gold:20
silver:12
bronze:13
total:45
-----
rank4:澳大利亚
gold:18
silver:19
bronze:16
total:53
-----
rank5:法国
gold:16
silver:26
bronze:22
total:64
-----
rank6:荷兰
gold:15
silver:7
bronze:12
total:34
-----
rank7:英国
gold:14
silver:22
bronze:29
total:65
-----
rank8:大韩民国
gold:13
silver:9
bronze:10
total:32
-----
rank9:意大利
gold:12
silver:13
bronze:15
total:40
-----
rank10:德国
gold:12
silver:13
bronze:8
total:33
-----
rank11:新西兰
gold:10
silver:7
bronze:3
total:20
-----
rank12:加拿大
gold:9
silver:7
bronze:11
total:27
-----
rank13:乌兹别克斯坦
gold:8
silver:2
bronze:3
total:13
-----
rank14:匈牙利
gold:6
silver:7
bronze:6
total:19
-----
rank15:西班牙
gold:5
silver:4
bronze:9
total:18
-----
rank16:瑞典
gold:4
silver:4
bronze:3
total:11
-----
rank17:肯尼亚
gold:4
silver:2
bronze:5
total:11
-----
rank18:挪威
gold:4
silver:1
bronze:3
total:8
-----
rank19:爱尔兰
gold:4
silver:0
bronze:3
total:7
-----
rank20:巴西
gold:3
silver:7
bronze:10
total:20
-----
rank21:伊朗伊斯兰共和国
gold:3
silver:6
bronze:3
total:12
-----
rank22:乌克兰
gold:3
silver:5
bronze:4
total:12
-----
rank23:罗马尼亚
gold:3
silver:4
bronze:2
total:9
-----
rank24:格鲁吉亚
gold:3
silver:3
bronze:1
total:7
-----
rank25:比利时
gold:3
silver:1
bronze:6
total:10
-----
rank26:保加利亚
gold:3
silver:1
bronze:3
total:7
-----
rank27:塞尔维亚
gold:3
silver:1
bronze:1
total:5
-----
rank28:捷克
gold:3
silver:0
bronze:2
total:5
-----
rank29:丹麦
gold:2
silver:2
bronze:5
total:9
-----
rank30:阿塞拜疆
gold:2
silver:2
bronze:3
total:7
-----
rank30:克罗地亚
gold:2
silver:2
bronze:3
total:7
-----
rank32:古巴
gold:2
silver:1
bronze:6
total:9
-----
rank33:巴林
gold:2
silver:1
bronze:1
total:4
-----
rank34:斯洛文尼亚
gold:2
silver:1
bronze:0
total:3
-----
rank35:Chinese Taipei
gold:2
silver:0
bronze:5
total:7
-----
rank36:奥地利
gold:2
silver:0
bronze:3
total:5
-----
rank37:菲律宾
gold:2
silver:0
bronze:2
total:4
-----
rank37:中国香港
gold:2
silver:0
bronze:2
total:4
-----
rank39:阿尔及利亚
gold:2
silver:0
bronze:1
total:3
-----
rank39:印度尼西亚
gold:2
silver:0
bronze:1
total:3
-----
rank41:以色列
gold:1
silver:5
bronze:1
total:7
-----
rank42:波兰
gold:1
silver:4
bronze:5
total:10
-----
rank43:哈萨克斯坦
gold:1
silver:3
bronze:3
total:7
-----
rank44:南非
gold:1
silver:3
bronze:2
total:6
-----
rank44:泰国
gold:1
silver:3
bronze:2
total:6
-----
rank44:牙买加
gold:1
silver:3
bronze:2
total:6
-----
rank47:埃塞俄比亚
gold:1
silver:3
bronze:0
total:4
-----
rank48:瑞士
gold:1
silver:2
bronze:5
total:8
-----
rank49:厄瓜多尔
gold:1
silver:2
bronze:2
total:5
-----
rank50:葡萄牙
gold:1
silver:2
bronze:1
total:4
-----
rank51:希腊
gold:1
silver:1
bronze:6
total:8
-----
rank52:阿根廷
gold:1
silver:1
bronze:1
total:3
-----
rank52:埃及
gold:1
silver:1
bronze:1
total:3
-----
rank52:突尼斯
gold:1
silver:1
bronze:1
total:3
-----
rank55:博茨瓦纳
gold:1
silver:1
bronze:0
total:2
-----
rank55:圣卢西亚
gold:1
silver:1
bronze:0
total:2
-----
rank55:乌干达
gold:1
silver:1
bronze:0
total:2
-----
rank55:智利
gold:1
silver:1
bronze:0
total:2
-----
rank59:多米尼加
gold:1
silver:0
bronze:2
total:3
-----
rank60:摩洛哥
gold:1
silver:0
bronze:1
total:2
-----
rank60:危地马拉
gold:1
silver:0
bronze:1
total:2
-----
rank62:巴基斯坦
gold:1
silver:0
bronze:0
total:1
-----
rank62:多米尼克
gold:1
silver:0
bronze:0
total:1
-----
rank64:土耳其
gold:0
silver:3
bronze:5
total:8
-----
rank65:墨西哥
gold:0
silver:3
bronze:2
total:5
-----
rank66:哥伦比亚
gold:0
silver:3
bronze:1
total:4
-----
rank66:亚美尼亚
gold:0
silver:3
bronze:1
total:4
-----
rank68:朝鲜民主主义人民共和国
gold:0
silver:2
bronze:4
total:6
-----
rank68:吉尔吉斯斯坦
gold:0
silver:2
bronze:4
total:6
-----
rank70:立陶宛
gold:0
silver:2
bronze:2
total:4
-----
rank71:印度
gold:0
silver:1
bronze:5
total:6
-----
rank72:摩尔多瓦共和国
gold:0
silver:1
bronze:3
total:4
-----
rank73:科索沃
gold:0
silver:1
bronze:1
total:2
-----
rank74:巴拿马
gold:0
silver:1
bronze:0
total:1
-----
rank74:斐济
gold:0
silver:1
bronze:0
total:1
-----
rank74:蒙古
gold:0
silver:1
bronze:0
total:1
-----
rank74:塞浦路斯
gold:0
silver:1
bronze:0
total:1
-----
rank74:约旦
gold:0
silver:1
bronze:0
total:1
-----
rank79:塔吉克斯坦
gold:0
silver:0
bronze:3
total:3
-----
rank80:阿尔巴尼亚
gold:0
silver:0
bronze:2
total:2
-----
rank80:波多黎各
gold:0
silver:0
bronze:2
total:2
-----
rank80:格林纳达
gold:0
silver:0
bronze:2
total:2
-----
rank80:马来西亚
gold:0
silver:0
bronze:2
total:2
-----
rank84:奥林匹克难民代表团
gold:0
silver:0
bronze:1
total:1
-----
rank84:佛得角
gold:0
silver:0
bronze:1
total:1
-----
rank84:卡塔尔
gold:0
silver:0
bronze:1
total:1
-----
rank84:科特迪瓦
gold:0
silver:0
bronze:1
total:1
-----
rank84:秘鲁
gold:0
silver:0
bronze:1
total:1
-----
rank84:斯洛伐克
gold:0
silver:0
bronze:1
total:1
-----
rank84:新加坡
gold:0
silver:0
bronze:1
total:1
-----
rank84:赞比亚
gold:0
silver:0
bronze:1
total:1
-----
中国VS美利坚合众国 霹雳舞
gold:0/0
silver:0/0
bronze:1/1
total:1/1
-----